docs/conf_sso_test_msscolab.rst

```diff
@@ -1,4 +1,4 @@
-Configuration MSS Colab Server with Testing IDP for SSO
+Configuration MSS Colab Server with Testing IdP for SSO
 =======================================================
 Testing IDP (`mslib/idp`) is designed specifically for testing the Single Sign-On (SSO) process using PySAML2.
 
```

Comment:
some lines below, 

`Since mscolab server’s path was set as the default path for the key and certificate, you should manually update the path of SERVER_CERT with the path of the generated .crt file for IDP, and SERVER_KEY with the path of the generated .key file for the IDP in the file MSS/mslib/idp/idp_conf.py`

Since devlopment this is an option. But when this should be an advice for someone to use a company cert and to learn where to store these files, we should always describe "the official" way. 

look from it from the perspective of an installed package.

Reply:
updated :)